### ISSUE TITLE
Dev dependency vs dependency

### ISSUE THREAD

**Issue by dem0nbruce** [TrilonIO/aspnetcore-Vue-starter#133]
Why are fontawesome,bootstrap, jquery, etc dev dependencies and not dependencies in package.json? Aren't they all client side dependencies? What am I missing here?

**Comment by MarkPieszak** [comment]
You're totally correct! I'm not sure when that shift might of happened actually but those should be regular dependencies. The project should work regardless, but I'll move those over to be safe. Thanks for pointing it out!
